pkg/handlers/troubleshoot.go

```diff
@@ -465,6 +469,49 @@ func (h *Handler) CollectSupportBundle(w http.ResponseWriter, r *http.Request) {
 	JSON(w, http.StatusAccepted, collectSupportBundlesResponse)
 }
 
+func (h *Handler) CollectHelmSupportBundle(w http.ResponseWriter, r *http.Request) {
+	appSlug := mux.Vars(r)["appSlug"]
+
+	if os.Getenv("IS_HELM_MANAGED") != "true" {
```

Comment:
could we please have this in a util function or something instead of always using `os.Getenv("IS_HELM_MANAGED")`? can be easily misspelled if you don't copy it.

Reply:
and if we wanna add more complex logic in the future to determine if it's helm managed or not